modules/virt-attaching-vm-to-primary-udn-web.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-attaching-vm-to-primary-udn-web_{context}"]                                  
+= Attaching a virtual machine to the primary user-defined network by using the web console
+
```

Comment:
We should make it clear at the top that by default it just works. That VMs created in a namespace where primary UDN is configured will automatically attach to it using the l2bridge network binding. Maybe that should be the first info in the procedure. And only then the optional switch to passt should follow